src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -691,7 +691,7 @@ public BitbucketTeam getTeam() throws IOException, InterruptedException {
         }
 
         final UriTemplate template = UriTemplate.fromTemplate(V2_API_REPOS + "{/owner}{?role,page,pagelen}")
```

Comment:
I forgot to change `{/owner}' to `{/ownerId}' in this commit. Will address this in the next commit

Reply:
If you have the time, please add some test for URI template expanding :sweat: Thank you! :baby_chick: